README.md

```diff
@@ -45,15 +44,9 @@ To get a local copy up and running follow these simple example steps.
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@bettercallsully(https://github.com/bettercallsully)
+- Twitter: [@bettrcallsully(https://twitter.com/bettrcallsully)
```

Comment:
[OPTIONAL]

- [ ] The README is the most important part of our Repository we should verify that it's content has the correct markdown format. Let's edit this lines and wrap `@bettercallsully` between `[` and `]`, to convert the GitHub user name into a link.